File: JAVA/Aksesoris.java
/*Saya Klara Ollivviera Augustine Gunawan dengan NIM 2306205 
mengerjakan soal Tugas Praktikum 2 dalam mata kuliah DPBO 
untuk keberkahanNya maka saya tidak melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin */
public class Aksesoris extends Petshop{//class Aksesoris extends kelas Petshop
    //atribut private
    private String jenis;
    private String bahan;
    private String warna;

    public Aksesoris(){//konstruktor
        super();
        this.jenis = "";
        this.bahan = "";
        this.warna = "";
    }

    public Aksesoris(String id, String nama_produk, int harga_produk, int stok_produk, String jenis, String bahan, String warna){
        super(id, nama_produk, harga_produk, stok_produk);
        this.jenis = jenis;
        this.bahan = bahan;
        this.warna = warna;
    }

    //getter dan setter
    public String get_jenis(){
        return this.jenis;
    }

    public void set_jenis(String jenis){
        this.jenis = jenis;
    }

    public String get_bahan(){
        return this.bahan;
    }

    public void set_bahan(String bahan){
        this.bahan = bahan;
    }

    public String get_warna(){
        return this.warna;
    }

    public void set_warna(String warna){
        this.warna = warna;
    }
}

File: PYTHON/Aksesoris.py
# Saya Klara Ollivviera Augustine Gunawan dengan NIM 2306205 
# mengerjakan soal Tugas Praktikum 2 dalam mata kuliah DPBO 
# untuk keberkahanNya maka saya tidak melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin

from Petshop import Petshop

class Aksesoris(Petshop):
    #atribut private
    __jenis = ""
    __bahan = ""
    __warna = ""

    #konstruktor
    def __init__(self, id = "", nama_produk = "", harga_produk = 0, stok_produk = 0, jenis = "", bahan = "", warna = ""):
        super().__init__(id, nama_produk, harga_produk, stok_produk)
        self.__jenis = jenis
        self.__bahan = bahan
        self.__warna = warna

    #getter dan setter
    def get_jenis(self):
        return self.__jenis
    
    def set_jenis(self, jenis):
        self.__jenis = jenis

    def get_bahan(self):
        return self.__bahan
    
    def set_bahan(self, bahan):
        self.__bahan = bahan

    def get_warna(self):
        return self.__warna
    
    def set_warna(self, warna):
        self.__warna = warna